kivakit-filesystems/java/src/test/java/com/telenav/kivakit/filesystem/java/JavaFileTest.java

```diff
@@ -0,0 +1,58 @@
+package com.telenav.kivakit.filesystem.java;
+
+import com.telenav.kivakit.test.UnitTest;
+import org.junit.Test;
+
+import java.nio.file.FileSystem;
+import java.nio.file.FileSystems;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.nio.file.Files;
+import java.nio.file.DirectoryStream;
+import java.nio.file.FileStore;
+import java.util.HashMap;
+import java.util.Map;
+
+public class JavaFileTest extends UnitTest {
+
+    @Test
+    public void testJavaFile()
+    {
+//        final var f = new JFSFile("/Users/yyzhou/text.zip");
+
+        Path zipfile = Paths.get("/Users/yyzhou/text.zip");
```

Comment:
This test looks to me like it tests Java and not our JavaFile class

Reply:
fixed